STYLE.md

```diff
@@ -649,6 +649,40 @@ A common mistake with refs is using them to pass data back to a parent component
 
 There are several ways to use and declare refs and we prefer the [callback method](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs).
 
+In addition, all refs should be declared in the constructor and null-initialized, rather than inline. This makes it easier to quickly see what refs are declared in the component:
+
+```jsx
+class BadRefComponent extends Component {
+    constructor(props) {
+        super(props);
+    }
+    
+    render() {
+        return (
+            <View
+                // Bad: Ref is declared inline
```

Comment:
I think this `Bad` belongs in the constructor? Otherwise it seems like we are implying it is wrong to set up a ref like this, but it's the correct way.

Reply:
Good point, moved to the constructor.